classifieds/classifieds-war/src/main/java/org/silverpeas/components/classifieds/servlets/ClassifiedsRequestRouter.java

```diff
@@ -90,6 +90,11 @@ public String getDestination(String function, ClassifiedsSessionController class
     request.setAttribute("CurrentFirstItemIndex", classifiedsSC.getCurrentFirstItemIndex());
 
     try {
+      boolean portletMode = "portlet".equalsIgnoreCase(function);
+      if (portletMode) {
+        function = "Main";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "function". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1226)